packages/edit-site/src/store/private-actions.js

```diff
@@ -62,9 +59,13 @@ export const setEditorCanvasContainerView =
  */
 export const setPageContentFocusType =
 	( pageContentFocusType ) =>
-	( { dispatch } ) => {
-		dispatch( {
-			type: 'SET_PAGE_CONTENT_FOCUS_TYPE',
-			pageContentFocusType,
-		} );
+	( { registry } ) => {
+		registry
```

Comment:
Should we deprecate this?

Reply:
We should remove it, it's a private action.